### DISCUSSION TITLE
UI/UX and product design

### DISCUSSION THREAD

**Discussion by JorchRL** [deleteman/the-hive#8] [Ideas]
I was wondering if the UI/UX of the hive is something that has already been planned. And maybe if I could help with that part. :sweat_smile:

**Comment by deleteman** [comment]
There is no UI/UX defined other than the one provided by the theme I'm using.
That said, I could definitely use some help defining the UX/UI. Right now I'm trying to get the repo details page going, and once I have a version of it, you're more than welcome to suggest or even create a better version of it with the data we have available.

As for the rest of the site, I do have some high-level ideas, but I'd love to know what your thoughts are!

**Reply by JorchRL** [reply]
I see! Can I present a proposal? 

Maybe I can contribute something meaningful, given what I understand from what you have shared. 

I just don't want to impose my own ideas :sweat_smile:

**Reply by deleteman** [reply]
Go ahead, please! You won't impose, any and all proposals are welcomed!

**Reply by JorchRL** [reply]
Thank you! I'll get to work on that! 

Do you mind if I write a blog post of it. Without revealing much details if you don't want to :sweat_smile: :

**Reply by deleteman** [reply]
Don't worry about it! Mention the repo if you'd like, let's spread the word!
